Makefile

```diff
@@ -0,0 +1,24 @@
+IMAGE_REGISTRY ?= quay.io
+IMAGE_ORG ?= strimzi
+IMAGE_REPO ?= strimzi-canary
+IMAGE_TAG ?= 0.0.1
```

Comment:
I think we should follow the names from other repositories:

```
DOCKER_REGISTRY    ?= docker.io
DOCKER_ORG         ?= $(USER)
DOCKER_TAG         ?= latest
```

Possibly with the same default values.